pycaret/time_series/forecasting/oop.py

```diff
@@ -1003,7 +1005,7 @@ def _setup_display_container(self) -> "TSForecastingExperiment":
         self.logger.info(f"Setup Display Container: {self.display_container[0]}")
         if self.verbose:
             pd.set_option("display.max_rows", 100)
-            print(self.display_container[0].style.apply(highlight_setup))
+            display(self.display_container[0].style.apply(highlight_setup))
```

Comment:
Can we use our Display class here?

Reply:
I am reusing the one from the regression oop file. Is that not the one to use?